### ISSUE TITLE
Add favorites UI to annotation browser

### ISSUE THREAD

**Issue by marktaipan-cvisionai** [cvisionai/tator#1520]
**Category**
Feature Request

**Problem**
Localization and state favorites can only be applied within the save dialog (i.e. at creation)

**Request**
- Add ability to pull up the user's favorites list within the annotation browser and allow being able to apply the favorite.
- Ideally this would also include media.
- Need to have undo work

**Comment by bctcvai** [comment]
With the new PR in-bound; one can add media_ids directly to a section. One could implement this PR by making a section called  "Favorites" and adding media_id to it. One could make "Other favorites" Section too. Suggest we collapse/close this once #1539 is merged.

**Comment by marktaipan-cvisionai** [comment]
Something like this would work:
![image](https://github.com/user-attachments/assets/6d356c47-1f65-4618-b6da-3e2de89e3c68)
